components/property_access.rst

```diff
@@ -394,6 +394,51 @@ and ``removeChild()`` methods to access to the ``children`` property.
 
 If available, *adder* and *remover* methods have priority over a *setter* method.
 
+Using non-standard adder/remover methods
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Like we just said, ``PropertyAccessor`` class allows to update the content of arrays stored in properties through
+*adder* and *remover* methods. But sometimes theses methods doesn't use the standard ``add`` and ``remove`` prefix::
+
+    // ...
+    class PeopleList
+    {
+        /**
+         * @var string[]
+         */
+        private $peoples = [];
+
+        public function getPeoples(): array
+        {
+            return $this->peoples;
+        }
+
+        public function joinPeople(string $people): void
+        {
+            $this->peoples[] = $people;
+        }
+
+        public function leavePeople(string $people): void
+        {
+            foreach ($this->peoples as $id => $item) {
+                if ($people === $item) {
+                    unset($this->peoples[$id]);
+                    break;
+                }
+            }
+        }
+    }
+
+    $list = new PeopleList();
+    $reflectionExtractor = new ReflectionExtractor(null, null, ['join', 'leave']);
+    $propertyAccessor = new PropertyAccessor(false, false, null, true, $reflectionExtractor, $reflectionExtractor);
+    $propertyAccessor->setValue($person, 'peoples', ['kevin', 'wouter']);
```

Comment:
Please add the necessary use statements for these classes (to make it more clear that they are in different components).